setup.py

```diff
@@ -20,7 +20,7 @@ def readme(short=False):
 
 setup(
     name="pyoscode",
-    version="1.0.2",
+    version="1.0.4",
```

Comment:
Is `setup.py` the only place that records the oscode version number? Or does that need updating somewhere else?

Reply:
The version number is recorded in `README.rst` and in `setup.py`.

Reply:
Wait, where in the README? Do you just mean the changelog?

Reply:
Yes, that's what I mean.